verification/helpers.go

```diff
@@ -1,13 +1,51 @@
 package verification
 
 import (
+	"encoding/json"
 	"fmt"
+	"os"
+	"path/filepath"
 	"regexp"
 	"strings"
 
 	ldapv3 "github.com/go-ldap/ldap/v3"
 )
 
+func loadPolicyDocument(policyDocumentPath string) (*PolicyDocument, error) {
+	policyDocument := &PolicyDocument{}
+	jsonFile, err := os.Open(policyDocumentPath)
+	if err != nil {
+		return nil, err
+	}
+	defer jsonFile.Close()
+	err = json.NewDecoder(jsonFile).Decode(policyDocument)
+	if err != nil {
+		return nil, err
+	}
+	return policyDocument, nil
+}
+
+func loadX509TrustStores(policyDocument *PolicyDocument, trustStoreBasePath string) (map[string]*X509TrustStore, error) {
+	var result = make(map[string]*X509TrustStore)
+	for _, trustPolicy := range policyDocument.TrustPolicies {
+		for _, trustStore := range trustPolicy.TrustStores {
```

Comment:
It seems this method loads all the trust stores references by any of the trust policies, why do we need to load all of then in the context of a single signature verification.

Reply:
This is to compute hashes on the trust store certificates so that we can early early during verification

From https://github.com/notaryproject/notaryproject/blob/main/signing-and-verification-workflow.md#verification-steps
> Calculate the SHA-256 fingerprint of all the trusted certificates and compare them against the list of SHA-256 certificate fingerprints present in io.cncf.notary.x509certs.fingerprint.sha256 annotation of artifact manifest.